app/views/users/index.html.erb

```diff
@@ -1 +1,15 @@
-<h1>This is details of users<h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;700&display=swap" rel="stylesheet">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  </head>
+
+  <body>
+    <div class="root-container users">
+      <% @users.each do |user| %>
+        <%= render "shared/user_info", user: user %>
+      <%end%>
+    </div>
```

Comment:
- Hi, Nice job so far. Can you make it possible for me to move to the show page when click on each user? Because the requirements say that: clicking on each user takes you to their show page as seen in the picture below ( moving from 1 to 2). 
- I navigated manually to see the user's show page which is not the project requirements.
![4](https://github.com/Paul-tes/Blog-App/assets/63972670/ba9a9779-6f57-4617-b19d-3b3488cd6494)
- Yours
![1](https://github.com/Paul-tes/Blog-App/assets/63972670/de383462-62bf-4435-9932-02730fbb1c80)